src/assets/items/items.ts

```diff
@@ -42,60 +42,67 @@ export const items: { [id: string]: Item } = {
         value: 5,
         type: ItemType.insant,
     },
-    Wood: {
+    wood: {
         id: "wood",
         name: "Wood",
         description: "Common Wood used in crafting",
         value: 5,
         type: ItemType.material,
     },
-    Pebble: {
+    pebble: {
         id: "pebble",
         name: "Pebble",
         description: "Small rock, used in crafting and tripping foes",
         value: 5,
         type: ItemType.material,
     },
-    Vine: {
+    vine: {
         id: "vine",
         name: "Vine",
         description: "Used in crafting",
         value: 25,
         type: ItemType.material,
     },
-    Leather: {
+    leather: {
         id: "leather",
         name: "Leather",
         description: "Used in crafting",
         value: 35,
         type: ItemType.material,
     },
-    Dandelion: {
+    dandelion: {
         id: "dandelion",
         name: "Dandelion",
         description: "Common weed with a puffy and yellow flower.",
         value: 35,
         type: ItemType.material,
     },
-    Fireweed: {
+    fireweed: {
         id: "fireweed",
         name: "Fireweed",
         description: "A bright pink flower often found by waters and roads.",
         value: 10,
         type: ItemType.material,
     },
-    MeadowButterCup: {
+    meadowbuttercup: {
         id: "meadowbuttercup",
-        name: "MeadowButterCup",
+        name: "Meadow ButterCup",
         value: 15,
         description: "A common myth is that meadow Buttercups are made of butter",
         type: ItemType.material,
     },
-    FourLeafedClover: {
-        id: "4leafclover",
-        name: "FourLeafedClover",
+    fourleafclover: {
+        id: "fourleafclover",
+        name: "4-Leafed Clover",
         value: 250,
         description: "You lucky motherfucker",
         type: ItemType.material,
     },
+    minecraftstevediamondpickaxe: {
+        id: "minecraftstevediamondpickaxe",
+        name: "Minecraft Steve's Diamond Picaxe",
+        value: 400,
+        description: "We don't need to stripmine",
+        type: ItemType.weapon,
+    },
```

Comment:
this is a weapon so it should be defined in the weapon item list. It also has no damage for a weapon.